### DISCUSSION TITLE
ZBBridge-U

### DISCUSSION THREAD

**Discussion by RastopchinSS** [arendst/Tasmota#22042] [Support request] [answered]
Добрый день!

Появилось новое устройство Sonoff Zigbee Bridge Ultra (ZBBridge-U).
Подскажите пожалуйста, есть ли у Вас прошивка для этого устройства, чтобы я мог подключить его к Home Assistant через интеграцию Zigbee2MQTT - ?
Если есть, могли бы Вы мне прислать инструкцию как это сделать?

**Comment by s-hadinger** [comment, marked as answer]
English please.

(translation)
A new device Sonoff Zigbee Bridge Ultra (ZBBridge-U) has appeared.
Please tell me if you have firmware for this device so that I can connect it to Home Assistant via Zigbee2MQTT integration - ?
If so, could you send me instructions on how to do this?

Answer: definitely no. Considering the specs of the Sonoff ZBBridge Ultra, it is not based on Espressif chip hence not supported by Tasmota. The price is also considerably higher, probably due to this new architecture.

**Comment by RastopchinSS** [comment]
It's clear. Thanks for the reply.
